### PR TITLE
Rename Elrond to MultiversX (configuration change)

### PR DESCRIPTION
**Pull request by andreibancioiu**
## Description

[`Elrond` is now `MultiversX`](https://twitter.com/beniaminmincu/status/1588162448103612416).

Applied the rename in `registry.json` and in a few comments (with links).

A PR against https://github.com/trustwallet/assets/ will follow.

## Checklist

- [ ] Create pull request as draft initially, unless its complete.
- [ ] Add tests to cover changes as needed.
- [ ] Update documentation as needed.
- [ ] If there is a related Issue, mention it in the description.

If you're adding a new blockchain

- [ ] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain.

### CODE REVIEW

**Comment by andreibancioiu** [comment]
@hewigovens, @catenocrypt, @Milerius, I've opened this PR instead of the previous one (https://github.com/trustwallet/wallet-core/pull/2857) - containing only the suggested change (along with some extra fixes within comments etc.).

File: docs/registry.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ This list is generated from [./registry.json](../registry.json)
 | 494     | BandChain        | BAND   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/band/info/logo.png" width="32" />         | <https://bandprotocol.com/>   |
 | 500     | Theta            | THETA  | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/theta/info/logo.png" width="32" />        | <https://www.thetatoken.org>  |
 | 501     | Solana           | SOL    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/solana/info/logo.png" width="32" />       | <https://solana.com>          |
-| 508     | Elrond           | eGLD   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/elrond/info/logo.png" width="32" />       | <https://elrond.com/>         |
+| 508     | MultiversX       | eGLD   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/multiversx/info/logo.png" width="32" />   | <https://multiversx.com/>     |
```
Comment (hewigovens) [inline docs/registry.md:52]:
```suggestion
| 508     | MultiversX       | eGLD   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/elrond/info/logo.png" width="32" />   | <https://multiversx.com/>     |
```

**Review by hewigovens — COMMENTED** [review]
looks good, left one comment

File: docs/registry.md (hunk 1 above)
Comment (hewigovens) [reply docs/registry.md:52]: New url is 404

File: src/Elrond/Codec.cpp [hunk 2]
```diff
@@ -25,7 +25,6 @@ std::string Codec::encodeBigInt(const std::string& value) {
     return encodeBigInt(uint256_t(value));
 }
 
-// For reference, see https://docs.elrond.com/developers/developer-reference/elrond-serialization-format/#arbitrary-width-big-numbers.
```
Comment (catenocrypt) [inline src/Elrond/Codec.cpp:28]: Can we just keep this comment for now?

File: tests/chains/Elrond/TWCoinTypeTests.cpp [hunk 3]
```diff
@@ -27,8 +27,8 @@ TEST(TWElrondCoinType, TWCoinType) {
     ASSERT_EQ(0x0, TWCoinTypeP2shPrefix(TWCoinTypeElrond));
     ASSERT_EQ(0x0, TWCoinTypeStaticPrefix(TWCoinTypeElrond));
     assertStringsEqual(symbol, "eGLD");
-    assertStringsEqual(txUrl, "https://explorer.elrond.com/transactions/1fc9785cb8bea0129a16cf7bddc97630c176a556ea566f0e72923c882b5cb3c8");
-    assertStringsEqual(accUrl, "https://explorer.elrond.com/address/erd12yne790km8ezwetkz7m3hmqy9utdc6vdkgsunfzpwguec6v04p2qtk9uqj");
+    assertStringsEqual(txUrl, "https://explorer.multiversx.com/transactions/1fc9785cb8bea0129a16cf7bddc97630c176a556ea566f0e72923c882b5cb3c8");
```
Comment (catenocrypt) [inline tests/chains/Elrond/TWCoinTypeTests.cpp:30]: This tx explorer link gives Not found (same with old elrond explorer), it should be changed to a valid TX ID.

**Review by catenocrypt — CHANGES_REQUESTED** [review]

File: tests/chains/Elrond/TWCoinTypeTests.cpp (hunk 3 above)
Comment (andreibancioiu) [reply tests/chains/Elrond/TWCoinTypeTests.cpp]: Indeed. Fixed: added a working example (also updated the _accounts_ URL).

File: src/Elrond/Codec.cpp (hunk 2 above)
Comment (andreibancioiu) [reply src/Elrond/Codec.cpp:28]: Added it. Note: `docs.multiversx.com` will become available in ~24-48 hours.

File: docs/registry.md (hunk 1 above)
Comment (andreibancioiu) [reply docs/registry.md]: Thank you! Fixed.

**Review by hewigovens — APPROVED** [review]

**Review by Milerius — APPROVED** [review]

**Review by catenocrypt — APPROVED** [review]

**Pull request merged**